samples/matter/light_bulb/boards/nrf54l15pdk_nrf54l15_cpuapp.conf

```diff
@@ -22,9 +22,8 @@ CONFIG_HWINFO_NRF=n
 
 CONFIG_NRF_802154_TEMPERATURE_UPDATE=n
 
-# TODO: Workaround that disables factory data until it will be supported.
-CONFIG_CHIP_FACTORY_DATA=n
-CONFIG_CHIP_FACTORY_DATA_BUILD=n
+# TODO: Workaround to be removed once Fprotect will be supported on nRF54L
```

Comment:
```suggestion
# TODO: Workaround to be removed once Zephyr's CONFIG_FPROTECT is supported on nRF54L
```
I think this gives a better idea when the action can be taken to remove this workaround...

Reply:
It is not only one workaround so far... :)